WooCommerce/Classes/ViewRelated/Orders/Order Creation/CollapsibleProductCard.swift

```diff
@@ -1,7 +1,87 @@
+
+    /// Tracks if discount editing should be enabled or disabled. False by default
+    var shouldDisableDiscountEditing: Bool = false
+
+    /// Tracks if discount should be allowed or disallowed.
+    var shouldDisallowDiscounts: Bool = false
+
+    @ScaledMetric private var scale: CGFloat = 1
+
+    init(viewModel: ProductRowViewModel,
+         flow: WooAnalyticsEvent.Orders.Flow,
+         shouldDisableDiscountEditing: Bool,
+         shouldDisallowDiscounts: Bool,
+         onAddDiscount: @escaping (Int64) -> Void) {
+        self.viewModel = viewModel
+        self.flow = flow
+        self.shouldDisableDiscountEditing = shouldDisableDiscountEditing
+        self.shouldDisallowDiscounts = shouldDisallowDiscounts
+        self.onAddDiscount = onAddDiscount
+    }
+
+    var body: some View {
+        if viewModel.childProductRows.isEmpty {
+            CollapsibleProductRowCard(viewModel: viewModel,
+                                      flow: flow,
+                                      shouldDisableDiscountEditing: shouldDisableDiscountEditing,
+                                      shouldDisallowDiscounts: shouldDisallowDiscounts,
+                                      onAddDiscount: onAddDiscount)
+            .overlay {
+                RoundedRectangle(cornerRadius: Layout.frameCornerRadius)
+                    .inset(by: 0.25)
+                    .stroke(Color(uiColor: .separator), lineWidth: Layout.borderLineWidth)
+            }
+        } else {
+            VStack(spacing: 0) {
+                // Parent product
+                CollapsibleProductRowCard(viewModel: viewModel,
+                                          flow: flow,
+                                          shouldDisableDiscountEditing: shouldDisableDiscountEditing,
+                                          shouldDisallowDiscounts: shouldDisallowDiscounts,
+                                          onAddDiscount: onAddDiscount)
+                Divider()
+                    .frame(height: Layout.borderLineWidth)
+                    .overlay(Color(.separator))
+
+                // Child products
+                ForEach(viewModel.childProductRows) { childRow in
+                    CollapsibleProductRowCard(viewModel: childRow,
+                                              flow: flow,
+                                              shouldDisableDiscountEditing: shouldDisableDiscountEditing,
+                                              shouldDisallowDiscounts: shouldDisallowDiscounts,
+                                              onAddDiscount: onAddDiscount) // TODO: #11250 Update design of child items in product bundle
+                    Divider().padding(.horizontal)
+                }
+            }
+            .overlay {
+                RoundedRectangle(cornerRadius: Layout.frameCornerRadius)
+                    .inset(by: 0.25)
+                    .stroke(Color(uiColor: .separator), lineWidth: Layout.borderLineWidth)
```

Comment:
nit: since this shares the same rounded rectangle with the other case in L42-44, maybe it can be extracted to a `@ViewBuilder` property for reuse